addons/quickmount/functions/fnc_addFreeSeatsActions.sqf

```diff
@@ -0,0 +1,218 @@
+#include "script_component.hpp"
+/*
+ * Author: Dystopian
+ * Creates actions for vehicle free seats.
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ * 1: Unit <OBJECT>
+ *
+ * Return Value:
+ * Child actions <ARRAY>
+ *
+ * Example:
+ * [cursorObject, player] call ace_quickmount_fnc_addFreeSeatsActions
+ *
+ * Public: No
+ */
+
+#define TO_COMPARTMENT_STRING(var) if !(var isEqualType "") then {var = format [ARR_2("Compartment%1",var)]}
+
+// this will check UAV vehicle like Stomper (with cargo seat)
+#define IS_CREW_UAV(vehicleConfig) ("UAVPilot" == getText (configFile >> "CfgVehicles" >> getText (vehicleConfig >> "crew") >> "simulation"))
+
+// workaround getting damage when moveout while vehicle is moving
+#define MOVEOUT_AND_BLOCK_DAMAGE \
+    if (isDamageAllowed _player) then { \
+        _player allowDamage false; \
```

Comment:
Might want to use
`[_player, "blockDamage", QUOTE(ADDON), true] call EFUNC(common,statusEffect_set);`

Reply:
`EFUNC(common,statusEffect_set)` would set global variable twice in 1 second. I don't think it's worth it to use it just for 1 second `allowDamage`.